### ISSUE TITLE
Glitch not working suddenly ??

### ISSUE THREAD

**Issue by outdoor720** [OmerTu/GoogleHomeKodi#300]
For some reason my glitch for Kodi stopped working, not sure what happened or changed

Here is the log when i try to run test connection
{
  "jsonrpc": "2.0",
  "id": 1,
  "method": "GUI.ShowNotification",
  "params": {
    "title": "GoogleHomeKodi",
    "message": "Test Successful!",
    "image": "info"
  }
}
request failed
route:  /koditestconnection
query:  {}
error:  { FetchError: request to http://65.94.136.91:8080/jsonrpc failed, reason: connect ETIMEDOUT 65.94.136.91:8080
    at ClientRequest.<anonymous> (/rbd/pnpm-volume/fce8aa25-b932-496e-ab1b-8942452d75ce/node_modules/.registry.npmjs.org/node-fetch/1.7.3/node_modules/node-fetch/index.js:133:11)
    at ClientRequest.emit (events.js:189:13)
    at Socket.socketErrorListener (_http_client.js:392:9)
    at Socket.emit (events.js:189:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
  name: 'FetchError',
  message:
   'request to http://65.94.136.91:8080/jsonrpc failed, reason: connect ETIMEDOUT 65.94.136.91:8080',
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT' }
body:  { token: 'xxxxx' }
Sending command to kodi http://65.94.136.91:8080/jsonrpc:
{
  "jsonrpc": "2.0",
  "id": 1,
  "method": "GUI.ShowNotification",
  "params": {
    "title": "GoogleHomeKodi",
    "message": "undefined: request to http://65.94.136.91:8080/jsonrpc failed, reason: connect ETIMEDOUT 65.94.136.91:8080",
    "image": "error"
  }
}
request failed
route:  /koditestconnection
query:  {}
error:  { FetchError: request to http://65.94.136.91:8080/jsonrpc failed, reason: connect ETIMEDOUT 65.94.136.91:8080
    at ClientRequest.<anonymous> (/rbd/pnpm-volume/fce8aa25-b932-496e-ab1b-8942452d75ce/node_modules/.registry.npmjs.org/node-fetch/1.7.3/node_modules/node-fetch/index.js:133:11)
    at ClientRequest.emit (events.js:189:13)
    at Socket.socketErrorListener (_http_client.js:392:9)
    at Socket.emit (events.js:189:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
  name: 'FetchError',
  message:
   'request to http://65.94.136.91:8080/jsonrpc failed, reason: connect ETIMEDOUT 65.94.136.91:8080',
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT' }
body:  { token: 'xxxxxx' }
Sending command to kodi http://65.94.136.91:8080/jsonrpc:
{
  "jsonrpc": "2.0",
  "id": 1,
  "method": "GUI.ShowNotification",
  "params": {
    "title": "GoogleHomeKodi",
    "message": "undefined: request to http://65.94.136.91:8080/jsonrpc failed, reason: connect ETIMEDOUT 65.94.136.91:8080",
    "image": "error"
  }
}

**Comment by keydon** [comment]
well glitch seems to be unable to reach your kodi.
Did your IP change? Changed firewall or port-forwarding rules?

**Comment by outdoor720** [comment]
> well glitch seems to be unable to reach your kodi.
> Did your IP change? Changed firewall or port-forwarding rules?

That's what i thought initially that my IP changed

I rechecked my IP, did not change.  I did not change my port forwarding or anything, but rechecked that as well.  All seems ok, so not sure why it is not working.

It was working great.  I got page down up working.  Nav left/right up down all working Nav left right # of times working

i'll keep looking at it and recheck again my ip, port forwarding etc
